### ISSUE TITLE
add_endpoint fails to add endpoint 242 for Green Power

### ISSUE THREAD

**Issue by konistehrad** [zigpy/zigpy-deconz#234]
The Conbee II on firmware 0x26780700 seems to only have two available endpoint slots, both of which are already consumed: one for ZHA (endpoint 1) and one for ZLL (endpoint 2). No idea where to go from here but I can't seem to get it to register 242.

**Comment by zigbeefordomoticz** [comment]
That is correct. There is a firmware which provides that functionality, but it looks like deConz never released it.
Maybe that is the case for conbeeIII

**Comment by konistehrad** [comment]
Yeah I was able to dig around in the [deconz-serial-protocol](https://github.com/dresden-elektronik/deconz-serial-protocol) issues and found a passing reference to the limitation. I've opened a new issue, and I guess we'll see where it goes. I think I'm starting to understand why ZGP support has been something of a uphill battle haha.

**Comment by zigbeefordomoticz** [comment]
You might want to try that one

https://github.com/zigbeefordomoticz/Domoticz-Zigbee/issues/1145#issuecomment-1160837120

**Comment by konistehrad** [comment]
Well look at you with the hookup! Yeah I’ll get this flashed and give it a shot ASAP. Thanks again.

**Comment by puddly** [comment]
deCONZ supports ZGP devices even with the two endpoint limitation so maybe it's not required to explicitly register one? I'll try to join a device to deCONZ and see what the serial traffic shows.

**Comment by konistehrad** [comment]
You still get direct Green Power Frame notifications, but I'm not able to get a Hue bulb to send me tunneled frames on 242 without it, at least not in zigpy.  I'd love to hear what Deconz is doing, and while you're in there please let me know if they're exposing 242 on the coordinator.
